app/common/notifications/notify.service.js

```diff
@@ -27,7 +27,7 @@ function Notify(_, $q, $rootScope, $translate, SliderService, ModalService) {
 
     function error(errorText, translateValues) {
         function showSlider(errorText) {
-            SliderService.openTemplate('<p>' + errorText + '</p>', 'warning', 'error');
```

Comment:
The error message was disappearing after the timeout, this seems problematic because by the time the user has half read and processed the issue it vanishes. It also means that they won't be able to grab error messages that they see and put them into issues.